src/Libraries/Microsoft.Extensions.Telemetry.Abstractions/buildTransitive/net6.0/Microsoft.Extensions.Telemetry.Abstractions.targets

```diff
@@ -1,16 +1,34 @@
 <Project>
-  <!-- This package should replace the Microsoft.Extensions.Logging.Abstractions source generator. -->
-  <Target Name="_Microsoft_Extensions_Logging_AbstractionsRemoveAnalyzers" 
+  <!-- This package should replace the "Microsoft.Extensions.Logging.Abstractions" source generator. -->
+  <Target Name="_Microsoft_Extensions_Logging_AbstractionsRemoveAnalyzers"
           Condition="'$(DisableMicrosoftExtensionsLoggingSourceGenerator)' == 'true'"
           AfterTargets="ResolveReferences">
     <ItemGroup>
-      <_Microsoft_Extensions_Logging_AbstractionsAnalyzer Include="@(Analyzer)" Condition="'%(Analyzer.AssemblyName)' == 'Microsoft.Extensions.Logging.Generators' Or
-                                                                                           '%(Analyzer.NuGetPackageId)' == 'Microsoft.Extensions.Logging.Abstractions'" />
+      <_Microsoft_Extensions_Logging_AbstractionsAnalyzer
+        Include="@(Analyzer)"
+        Condition="'%(Analyzer.AssemblyName)' == 'Microsoft.Extensions.Logging.Generators' Or
+                   '%(Analyzer.NuGetPackageId)' == 'Microsoft.Extensions.Logging.Abstractions'" />
     </ItemGroup>
 
     <!-- Remove Microsoft.Extensions.Logging.Abstractions Analyzer -->
     <ItemGroup>
       <Analyzer Remove="@(_Microsoft_Extensions_Logging_AbstractionsAnalyzer)" />
     </ItemGroup>
   </Target>
+
+  <!-- We have a separate target for WPF projects because there's "GenerateTemporaryTargetAssembly" target for those -->
```

Comment:
The comment is a good idea, but I feel like it could be expanded a little to make the reason clearer.

Reply:
Any suggestions?

Reply:
I'd summarise the information from the discussion above so won't need to scratch our heads two months from now wondering why this target is needed. So, I'd make the comment something like this:

_We have a separate target for WPF projects because generates a generates and compiles a temporary project, and that temp project doesn't have "ResolveReferences" we expect. This leads to the runtime's generator not being removed and to a compile-time failure. This target resolves the problem._

Reply:
Thanks for the suggestion, applied it